docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Typos below:</p><ul><li>... tempering and a<strong>n</strong> MCMC kernel.</li><li>... We use Nt, where t is the threshold parameter -- <strong>by default t=0.5. This means that the default ESS is fixed at half the number of draws.</strong></li><li>"Generate N samples Sβ from the prior...": is the number of samples N or S_beta? I think N is the number of MCMC chains, isn't it ?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>